elements/designer-element-palette/designer-element-palette.html

```diff
@@ -64,10 +59,16 @@
     <h1>Elements</h1>
     <div id="elements">
       <template is="dom-repeat" items="{{builtIns}}">
-        <div class="element" on-mousedown="_onMouseDown">&lt;<span>{{item}}</span>&gt;</div>
+        <div class="element" on-mousedown="_onMouseDown">
```

Comment:
I'm thinking less about the usability of designer and more about just using Polymer, since it's the tool at our disposal.